catalog/camel-catalog/src/generated/resources/org/apache/camel/catalog/components.properties

```diff
@@ -175,7 +175,6 @@ ignite-queue
 ignite-set
 imap
 imaps
-infinispan
```

Comment:
I think we need a special case for infinispan in the catalog generation

Reply:
Yeah if you have a sub folder, like camel-salesforce etc then you need to do some code changes in the tooling, you can do a git grep salesforce and find where it does some special.

Over time we may want to improve the tooling so it can "handle this". For example if we move all the AWS components into sub folder.